src/resolvers/feedback.ts

```diff
@@ -0,0 +1,319 @@
+            throw NotFoundError(`Plan with ID ${planId} not found`);
+          }
+
+          // Check that user has permissions to access feedback
+          const projectId = plan.projectId;
+          const project = await Project.findById(reference, context, projectId);
+          if (!project) {
+            throw NotFoundError(`Project with ID ${projectId} not found`);
+          }
+          if (await hasPermissionOnProject(context, project)) {
+            const temp = await PlanFeedback.findByPlanId(reference, context, planId);
+            console.log("****PLAN FEEDBACK", temp);
+            return temp;
+          }
+        }
+        throw context?.token ? ForbiddenError() : AuthenticationError();
+      } catch (err) {
+        if (err instanceof GraphQLError) throw err;
+
+        context.logger.error(prepareObjectForLogs(err), `Failure in ${reference}`);
+        throw InternalServerError();
+      }
+    },
+
+    // Get all of the comments associated with the round of admin feedback
+    planFeedbackComments: async (_, { planId, planFeedbackId }, context: MyContext): Promise<PlanFeedbackComment[]> => {
+      const reference = 'feedback resolver';
+      try {
+        if (isAuthorized(context.token)) {
+          // Check to see if planId exists in our records
+          const plan = await Plan.findById(reference, context, planId);
+          if (!planId) {
+            throw NotFoundError(`Plan with ID ${planId} not found`);
+          }
+
+          // Check that user has permissions to access feedback
+          const projectId = plan.projectId;
+          const project = await Project.findById(reference, context, projectId);
+          if (!project) {
+            throw NotFoundError(`Project with ID ${projectId} not found`);
+          }
+          if (await hasPermissionOnProject(context, project)) {
+            return await PlanFeedbackComment.findByFeedbackId(reference, context, planFeedbackId);
+          }
+        }
+        throw context?.token ? ForbiddenError() : AuthenticationError();
+      } catch (err) {
+        if (err instanceof GraphQLError) throw err;
+
+        context.logger.error(prepareObjectForLogs(err), `Failure in ${reference}`);
+        throw InternalServerError();
+      }
+    },
+  },
+
+  Mutation: {
+    //Request a round of admin feedback
+    requestFeedback: async (_, { planId }, context: MyContext): Promise<PlanFeedback> => {
+      const reference = 'requestFeedback resolver';
+
+      try {
+        if (isAuthorized(context.token)) {
+          const plan = await Plan.findById(reference, context, planId);
+          if (!plan) {
+            throw NotFoundError(`Plan with ID ${planId} not found`);
+          }
+          const project = await Project.findById(reference, context, plan.projectId);
+          if (!project) {
+            throw NotFoundError(`Project with ID ${plan.projectId} not found`);
+          }
+
+          if (await hasPermissionOnProject(context, project)) {
```

Comment:
maybe a question for @bofstein.

The current tool only allows the owner of the plan (or a co-owner) to request feedback. We are allowing anyone with edit access to do so here. Do we want that or should we restrict it to owner(s)?